data-raw/process_data_sets.R

```diff
@@ -275,6 +275,8 @@ usethis::use_data(pennies_resamples, overwrite = TRUE)
 
 # Massachussets 2020 vs. 2019 Traffic Data
 ma_traffic_2020_vs_2019 <- 
-  read_csv("data-raw/Massachusetts_2020_vs_2019_Traffic_Data.csv") %>% 
-  janitor::clean_names()
+  "data-raw/Massachusetts_2020_vs_2019_Traffic_Data.csv" %>% 
+  read_csv() %>% 
+  janitor::clean_names() %>% 
```

Comment:
You can remove the explicit call to `janitor::` as we load it at the start of the script